sam-hiberry-app/tests/events/shopify_event.json

```diff
@@ -0,0 +1,438 @@
+{
+    "version": "0",
+    "id": "a575d0e8-2e37-0b37-b1f5-29e8be49566f",
+    "detail-type": "shopifyWebhook",
+    "source": "aws.partner/shopify.com/71471267841/ShopifySourceTest",
```

Comment:
if this is for test, we could anonymize some of the values

Reply:
fields anonymized